### PR TITLE
fix console warning while using inputs on login page

### PR DESCRIPTION
**Pull request by mvoloshin0458**

### CODE REVIEW

File: src/components/inputs/LoginField/LoginField.tsx [hunk 1]
```diff
@@ -1,47 +1,53 @@
...
   label: string | ReactElement,
   type?: string,
-  tooltip?: any
-  error?: any
+  tooltip?: any,
+  error?: any,
 }
 function LoginField(props: LoginFieldProps) {
-  const { label, onChange, value, error, placeholder, type, wrapperClass = '', tooltip, isRequired, ...rest } = props
+  const { label, onChange, value = EMPTY_STRING, error, placeholder, type, wrapperClass = '', tooltip, isRequired, ...rest } = props
```
Comment (mvoloshin0458) [inline src/components/inputs/LoginField/LoginField.tsx:22]: wrapperClass = EMPTY_STRING

File: src/components/inputs/LoginField/LoginField.tsx [hunk 2]
```diff
@@ -1,47 +1,53 @@
...
+  onChange: (newVal: string | number) => void,
+  value?: string | number,
   isRequired?: boolean,
   wrapperClass?: string,
   placeholder?: string,
   label: string | ReactElement,
   type?: string,
-  tooltip?: any
-  error?: any
+  tooltip?: any,
```
Comment (mvoloshin0458) [inline src/components/inputs/LoginField/LoginField.tsx:18]: remove coma

File: src/components/inputs/LoginField/LoginField.tsx [hunk 3]
```diff
@@ -1,47 +1,53 @@
...
+        ? event.target.valueAsNumber
+        : event.target.value
+    )
   }
 
   const wrapperClasses = classNames({
     [wrapperClass]: true,
     'login-text-field': true,
-    'has-error': !!error
+    'has-error': !!error,
```
Comment (mvoloshin0458) [inline src/components/inputs/LoginField/LoginField.tsx:37]: remove coma

File: src/hooks/useToggle.ts [hunk 4]
```diff
@@ -1,12 +1,25 @@
...
+  initialState: string | boolean,
+  options?: string[] | undefined
+): [string | boolean, () => void] {
+  const [value, setValue] = useState<string | boolean>(initialState)
   const toggle = useCallback(() => {
     setValue((state) => {
-      if(options) {
-        const foundOption = options.find((option) => option !== state)
-        return foundOption || ''
+      if (options) {
```
Comment (mvoloshin0458) [inline src/hooks/useToggle.ts:17]: return

**Review by sofa-tata — APPROVED** [review]

**Pull request merged**